puppet/zulip/files/postgresql/process_fts_updates

```diff
@@ -43,6 +43,8 @@ sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), "../.
 try:
     os.environ['DJANGO_SETTINGS_MODULE'] = 'zproject.settings'
     from django.conf import settings
+    if settings.USING_PGROONGA:
+        exit()
```

Comment:
The deferred index update usually only adds <10ms of delay to the data being available in the index -- it just avoids blocking the critical `send_message` code path on updating the index.  

I definitely understand the desire to get something finished and merged and then move onto building improvements and optimizations.  I'm OK with fixing this being a follow-up issue, but I consider it a blocker for PGroonga being available in Zulip in production, because I don't want to support two models here (and I think it'll actually be very little work to make the PGroonga integration use the existing infrastructure for doing deferred updates we have for postgres full text search).